scripts/Makefile.lib

```diff
@@ -282,9 +282,15 @@ cmd_dt_S_dtb=						\
 $(obj)/%.dtb.S: $(obj)/%.dtb
 	$(call cmd,dt_S_dtb)
 
+ifneq ($(wildcard $(PROJECT_BASE)/$(BOARD_NAME).overlay),"")
```

Comment:
Do we need this test? The code below tests for the file's existence, so why not just set it? Perhaps

    DTS_OVERLAY ?= $(PROJECT_BASE)/$(BOARD_NAME).overlay

setting it weakly to make it easier to override.